### ISSUE TITLE
Navigation Back on XBox 

### ISSUE THREAD

**Issue by TrevorBarstowTT** [Windows-XAML/Template10#1270]
Using Template 10 Hamburger template app

When running the app on XBox there is no back key displayed, in the same manner as Mobile device family. Perhaps it should be? Note that XBox apps have 'mouse' pointer that can be turned on for UWP apps.

Using the app, back navigation is meant to be actioned by hitting 'B' key on XBox but this has no effect.
I have tried adding BackButtonBehavior to the (detail) page on this app but this is not fired.

In contrast the sample Template 10 Master Detail app on XBox does respond to the 'B' button via a BackButtonBehavior or normally.

So putting aside the fact that a Back button would be handy to be displayed on XBox, I'm guessing something needs to be added to the standard app template to enable Back Button to work on XBox.

Any ideas?

**Comment by darenm** [comment]
So in a "vanilla" UWP app running on Xbox, as long as you have the system back requested hooked up:

`SystemNavigationManager.GetForCurrentView().BackRequested += NavigationService_BackRequested;`

And you are **not** handling the Gamepad B request, the system back requested will be raised. The following code is an example of code that would stop the system back from being raised:

```
        private void HaveStarz_KeyUp(object sender, Windows.UI.Xaml.Input.KeyRoutedEventArgs e)
        {
            switch (e.OriginalKey)
            {
                case VirtualKey.GamepadB:
                    if (ViewModel.StartActivation)
                    {
                        ViewModel.NavigateFromActivation();
                        e.Handled = true;
                    }
                    break;
            }
        }
```

You will note the code `e.Handled = true;` - that prevents the B press from bubbling up and firing the  global back action.

**Comment by TrevorBarstowTT** [comment]
I spent a bit of time debugging this on XBox
1. 'B' key does not cause navigation back from the Detail Page of the standard Hamburger Template. This I think is because there are no controls to take focus on that page
2.'B' when pressed twice does navigate back from Settings Page of the standard Hamburger Template.

It was difficult to debug because connection kept getting lost during remote debugging session.

I note that Hamburger Menu does handle key down events including VirtualKey.GamepadB
There is some code in 

```
        private void BackHandler(object sender, BackRequestedEventArgs args)
        {
            DebugWrite();

            var handled = false;
            if (ApiInformation.IsApiContractPresent(nameof(Windows.Phone.PhoneContract), 1, 0))

```

which does not get executed on a mobile device because nameof gives "PhoneContract" not "Windows.Phone.PhoneContract". However back does seem to be processed on Phone anyway.

That's all I've found out.
